kinbot/kb.py

```diff
@@ -126,15 +130,13 @@ def main():
         logging.error('Error with initial structure optimization.')
         return
     if any(well0.freq[i] <= 0 for i in range(len(well0.freq))):
-        logging.error('Found imaginary frequency for initial structure.')
-        return
+        logging.error('Found imaginary frequency (freq # {}, value = {}) for initial structure.'.format(i, well0.freq[i]))
 
     # characterize again and look for differences
     well0.characterize(dimer=par['dimer'])
     well0.name = str(well0.chemid)
     if well0.name != start_name:
-        logging.error('The first well optimized to a structure different from the input.')
-        return
+       logging.error('The first well optimized to a structure different from the input.')
```

Comment:
Again, i let the calculation continue here, but noting the error in the log file.